manifest.toml

```
@@ -99,5 +99,5 @@ ram.runtime = "50M"
     packages = "deb1, deb2, php8.0-foo, php8.0-bar"
```

```suggestion
    packages = "mariadb-server, deb1, deb2, php8.0-foo, php8.0-bar"
```
I am wondering if we need to put MariaDB-server as a package to install as the database is set to MySQL...